File: api/code-samples/javascript/klay/block/getBlockWithConsensusInfoByNumber.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new Caver.ApiClient("https://api.baobab.klaytn.net:8651"));

    const blockNumber = 1;

    sdk.klay.getBlockWithConsensusInfoByNumber(blockNumber, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: api/code-samples/python/klay/block/getBlockWithConsensusInfoByNumber.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockTag = "0x6e0431"

sdk = OpenSDK(host)
klay_response = sdk.klay.get_block_with_consensus_info_by_number(blockTag)

print(json.loads(klay_response.response.data))
